<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>177</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>471</width>
    <height>177</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>471</width>
    <height>177</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Eliminar Curso</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(18, 82, 200);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_regresar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>141</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(18, 82, 200);
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;

font: 75 11pt &quot;Arial Black&quot;;
	
}

QPushButton:hover{
background-color: white;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;

font: 75 11pt &quot;Arial Black&quot;;
}</string>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pngfind.com-back-png-1934895.png</normaloff>pngfind.com-back-png-1934895.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>150</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>28</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

font: 75 12pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Codigo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_codigo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>25</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>400</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

font: 75 12pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>ingrese el curso a eliminar</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_eliminar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>141</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(18, 82, 200);
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;

font: 75 11pt &quot;Arial Black&quot;;
	
}

QPushButton:hover{
background-color: white;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;

font: 75 11pt &quot;Arial Black&quot;;
}</string>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>1492790846-9cancel_84247.png</normaloff>1492790846-9cancel_84247.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
